src/utils/script-mime-types.ts

```diff
@@ -10,15 +10,15 @@ const wrappingQuotesRe: RegExp = /(^("|'|`))|(("|'|`)$)/g;
 // https://iana.org/assignments/media-types/media-types.xhtml
 // Note: Don't need to put any suffixed types (+json, +xml) in here as it
 //       will be handled separately
-const scriptTypeToParserMap: Map<string, BuiltInParserName> = new Map([
-	['application/ecmascript', 'babel'],
-	['application/javascript', 'babel'],
-	['application/json', 'json'],
-	['text/ecmascript', 'babel'],
-	['text/javascript', 'babel'],
-	['text/markdown', 'markdown'],
-	['text/typescript', 'typescript']
-]);
```

Comment:
@OiYouYeahYou Should we also add `'module'` explicitly? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#applying_the_module_to_your_html